docs/source/devinstall.md

```diff
@@ -73,7 +86,7 @@ otherwise the command will use the default environment.
 Build a wheel file that can then be installed via `pip install`
 
 ```
-make dist
```

Comment:
Make dist will do all, do we need to call `bdist` explicitly here?

Reply:
@lresende This section is for dev workflow.  I figured `bdist` was more useful since devs typically just build the wheel file when iterating.  The description needs to change to include what the target does if we left it as `dist` and we'd want to remove the next section (for building the kernelspecs).

I added a fully explained `make dist` section a couple items below.